sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -451,7 +451,9 @@ def plan_to_named_target(self, name, custom_start_state=None):
         else:
             self._move_group_commander.set_start_state(custom_start_state)
         if self.set_named_target(name):
-            self.__plan = self._move_group_commander.plan()
+            self.__plan = self._move_group_commander.plan()[1]
+        else:
+            rospy.logwarn("Named target does not exist")
```

Comment:
maybe "failed to set to named target" is more accurate?

Reply:
Changed